yellowbrick/features/jointplot.py

```diff
@@ -42,9 +42,9 @@ class JointPlotVisualizer(FeatureVisualizer):
         This is inherited from FeatureVisualizer but is defined within
         JointPlotVisualizer since there are three axes objects.
 
-    feature: string, default: None
+    label: string, default: None
         The name of the X variable
-        If a DataFrame is passed to fit and feature is None, feature
+        If a DataFrame is passed to fit and label is None, label
```

Comment:
This is another name, where the relationship between label and feature doesn't quite match up. The argument `feature` here refers to the column of the `X` matrix, in direct contrast to the `target` which is actually a label in this case! In this case, I'd recommend changing `feature` to `column`. 

See #434 (which I just created to clarify how to change this visualizer)